src/pages/About.jsx

```diff
@@ -58,7 +67,13 @@ function About() {
         Reprehenderit enim aute commodo cupidatat ut ipsum ea minim magna velit.
       </p>
 
-      <h2>How UndocuScholar Works</h2>
+      <h2>
+        How
+        {' '}
+        {constants.brandName}
+        {' '}
+        Works
```

Comment:
Can't this just be `How {constants.brandName} Works`?

Reply:
I had it like that but the lint --fix changes it to that format. I'll change it back.

Reply:
should i disable 'react/jsx-one-expression-per-line', if not I get a lint error

Reply:
Hmm, if we do that it might make HTML subelements harder to read later. Maybe instead in `package.json` in the `eslintConfig` section you can add this:
```json
    "rules":{
      "quotes": ["error", "single", { "allowTemplateLiterals": true }]
    }
```
And then use a template here to make this read more easily and occupy less lines:
```js
{`How {constants.brandName} Works`}
```